config/query/format_boosting.yml

```diff
@@ -0,0 +1,49 @@
+government_index:
```

Comment:
It is important to point out that as a result of this change new Government formats will now need to be manually added to this list or they won't receive the boost they require. Given this is unlikely this will be remembered are we sure this is the correct way change to make.